packages/blockchain-common/package-lock.json

```diff
@@ -14,12 +14,12 @@
 			}
 		},
 		"@babel/generator": {
-			"version": "7.8.4",
-			"resolved": "https://registry.npmjs.org/@babel/generator/-/generator-7.8.4.tgz",
-			"integrity": "sha512-PwhclGdRpNAf3IxZb0YVuITPZmmrXz9zf6fH8lT4XbrmfQKr6ryBzhv593P5C6poJRciFCL/eHGW2NuGrgEyxA==",
+			"version": "7.8.8",
```

Comment:
these changes aren't needed as you haven't changed the package.json for blockchain-common